.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Release
+
+# Only trigger this action when a new release is published
+on:
+  release:
+    types: [published]
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      # Setup the workflow to use the specific version of Go
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.13.1'
+      # Checkout the repository
+      - name: Checkout
+        uses: actions/checkout@v2
+      # Cleans up unreachable Git objects and convert a shallow clone to a regular one if needed
+      - name: Unshallow
+        run: git fetch --prune --unshallow
+      # Run tests prior to release    
+      - name: Test
+        run: make test
+      # Publish the binaries
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v2
```

Comment:
Does this thing also end up creating tags and uploading bits to github releases?

Reply:
@stianst that tag is created by us when, after drafting a new release, we publish it. GoReleaser will be responsible to buildm publish the binaries and generate the changelog.

Reply:
@stianst The only manual steps you need to do is to `Draft a new release` and once you do that, you have a new tag and need to wait for the binaries to be published.

Tagging after the release is something we could improve and explore, but I don't see any other way of doing this, if we want to just press a button. Another alternative, would be to do that through the command line (more error prone).